spec/controllers/settings_controller_spec.rb

```diff
@@ -40,4 +40,16 @@
       expect(assigns(:settings).email).to eq attrs[:email]
     end
   end
+
+  describe "#regen_key" do
+    before do
+      sign_in user
+      @current_key = user.api_key
```

Comment:
I would throw this in a let statement instead of an ivar.

Outside of the before block:

`let(:current_key) { user.api_key }`

This is still correct though!